docs/changes.rst

```diff
@@ -172,6 +167,9 @@ Other changes:
   - re-use existing checkpoint functionality
   - archiver tests: add check_cache tool - lints refcounts
 
+- fixed cache sync performance regression from 1.1.0b1 onwards, #1940
+- syncing the cache without chunks.archive.d (see :ref:`disable_archive_chunks`)
+  now avoids any merges and is thus faster, #1940
```

Comment:
did you go back from $now until you found something from the git changelog already mentioned in this changes update?

as preparing this change took quite some days, the stuff of some last might be all missing.